src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Wasi.Sdk/Sdk/Sdk.props

```diff
@@ -3,7 +3,7 @@
     <TargetArchitecture>wasm</TargetArchitecture>
     <TargetOS>wasi</TargetOS>
     <UseMonoRuntime>true</UseMonoRuntime>
-    <OutputType>Exe</OutputType>
+    <!-- <OutputType>Exe</OutputType> -->
```

Comment:
It should be fine to leave the default here. The user's csproj is imported after this file and thus it will be overriden

Reply:
Actually it is overriding user's specified `OutputType`. 
I've updated this to check for `<OutputType Condition="'$(OutputType)' == ''">Exe</OutputType>`